### PR TITLE
feature/action state model

### LINKED ISSUE #16: Implement Cloudstate Action State Model
Implement the Cloudstate Actions Service from /protobuf/protocol/cloudstate/action.proto

WIP:
https://github.com/cloudstateio/go-support/tree/feature/action_state_model

- [x] Passing TCK
- [x] API refinement

### PR DESCRIPTION
**Pull request by marcellanz**
Fixes #16
- [x] Pass the TCK
- [x] API cleanup

### CODE REVIEW

**Comment by marcellanz** [comment]
I'm going to merge this PR soon to get on with #53.

File: cloudstate/cloudstate.go [hunk 1]
```diff
@@ -74,6 +78,17 @@ func (cs *CloudState) RegisterCRDT(entity *crdt.Entity, config protocol.Descript
 	return nil
 }
 
+// RegisterCRDT registers a CRDT entity.
```
Comment (pvlugter) [inline cloudstate/cloudstate.go:81]: Comment is for CRDT entity

**Review by pvlugter — COMMENTED** [review]
:rocket: 🥳

File: cloudstate/cloudstate.go (hunk 1 above)
Comment (marcellanz) [reply cloudstate/cloudstate.go:81]: Great catch, thanks for reading Go code!
Fixed with: https://github.com/cloudstateio/go-support/pull/64

**Pull request merged**